packages/ui/package.json

```diff
@@ -15,7 +15,7 @@
     "stylelint": "stylelint \"src/**/*.scss\"",
     "test": "jest"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
Why do we change these to `devDependencies`?

Reply:
you don't need these deps to run the spa, right? ;-)

Reply:
hmmm, I don't quite get it... Doesn't `ui` a standalone react app? Why doesn't it need these deps?

Reply:
parcel will bundle all the dependencies to `dist`

Reply:
Actually, I was thinking,  since we only package dist files, should we leave the dependencies settings as its default behavior.  Still, it is friendly to the individual developers.

Reply:
And we can remove `node_modules` in console after build.

Reply:
@demonzoo i'm lost, the purpose of this is to clean up unnecessary deps. so we need to clear all deps after building a frontend package. if we leave them in `dependencies` instead of `devDependencies`, things like `react` and `react-dom` will still sit inside `node_modules`, which is not expected

Reply:
@simeng-li @wangsijie the tricky part is, since we're using monorepo, we cannot simply clean up sub-package's `node_modules` since they are still in project root's `node_modules`

Reply:
please see https://github.com/facebook/create-react-app/issues/10219 and this [stackoverflow](https://stackoverflow.com/questions/44868453/create-react-app-install-devdepencies-in-dependencies-section/44872787#44872787)

Reply:
Got it, then we have to move them to `devDependencies`.

Reply:
resolve this since we're aligned